_includes/share-mentor.html

```diff
@@ -2,6 +2,7 @@
 {% assign mentor_name = mentor.name %} 
 {% assign full_url = base_url | append: '/mentors?keywords=' | append: mentor_name %}
 
+<div class="mentor-container">
```

Comment:
could you add screenshot with the bug and after with the fix? It is part of the TODO for collaborators.